src/pages/_app.tsx

```diff
@@ -14,12 +14,17 @@ import { ConnectKitProvider } from 'connectkit'
 import { connectKitTheme, wagmiClient } from '@utils/wallet'
 import AutomationProvider from '../@context/Automation/AutomationProvider'
 import { FilterProvider } from '@context/Filter'
+import Script from 'next/script'
 
 function MyApp({ Component, pageProps }: AppProps): ReactElement {
   Decimal.set({ rounding: 1 })
 
   return (
     <>
+      <Script
+        data-domain="portal.pontus-x.eu"
+        src="https://plausible.io/js/script.js"
+      />
```

Comment:
suggestion: Put the `data-domain` in a config file and only load the script if it's defined.